logbook-servlet/src/main/java/org/zalando/logbook/servlet/RemoteRequest.java

```diff
@@ -73,14 +75,22 @@ public String getRemote() {
 
     @Override
     public String getRequestUri() {
-        final String uri = getRequestURL().toString();
-        @Nullable final String queryString = getQueryString();
-        return queryString == null ? uri : uri + "?" + queryString;
+        return getRequestURL().toString();
+    }
+
+    @Override
+    public ListMultimap<String, String> getQueryParameters() {
+        final ListMultimap<String, String> parameters = ArrayListMultimap.create();
+        
+        getParameterMap().forEach((name, values) -> 
```

Comment:
> Unfortunately this would include POST parameters and the servlet api is not low-level enough to distinguish between query string and body parameters.

You're right. We could check for the method and assume that nobody is using query and form parameters at the same time. Or we openly discourage the use of `application/x-www-form-urlencoded` e.g. in the readme.